feature/search/src/main/kotlin/uk/govuk/app/search/ui/SearchScreen.kt

```diff
@@ -124,63 +133,90 @@ private fun SearchScreen(
 }
 
 @Composable
-fun ShowResults(searchResults: List<Result>, onClick: (String, String) -> Unit) {
-    Column(
-        modifier = Modifier
-            .padding(bottom = GovUkTheme.spacing.medium)
-            .fillMaxSize(),
-        horizontalAlignment = Alignment.CenterHorizontally
+fun ShowResults(
+    searchTerm: String,
+    searchResults: List<Result>,
+    onClick: (String, String) -> Unit
+) {
+    val listState = rememberLazyListState()
+    val focusRequester = remember { FocusRequester() }
+
+    var previousSearchTerm by rememberSaveable { mutableStateOf("") }
+
+    LazyColumn(
+        modifier = Modifier.fillMaxWidth(),
+        state = listState
     ) {
-        LazyColumn {
-            items(searchResults) { searchResult ->
-                val title = StringUtils.collapseWhitespace(searchResult.title)
-                val url = StringUtils.buildFullUrl(searchResult.link)
-
-                val context = LocalContext.current
-                val intent = Intent(Intent.ACTION_VIEW)
-                intent.data = Uri.parse(url)
-
-                GovUkCard(
-                    modifier = Modifier.padding(
-                        GovUkTheme.spacing.medium,
-                        GovUkTheme.spacing.medium,
-                        GovUkTheme.spacing.medium,
-                        0.dp
-                    ),
-                    onClick = {
-                        onClick(title, url)
-                        context.startActivity(intent)
-                    }
+        item {
+            Title3BoldLabel(
```

Comment:
Judging by this PR https://github.com/alphagov/govuk-mobile-android-app/pull/155 we might want to check if this should be set as a semantic header as I think it fits the criteria?

Reply:
Good shout!

Reply:
Done